src/index.js

```diff
@@ -0,0 +1,27 @@
+import './styles/main.scss';
+import Tasks from './modules/ClassStore.js';
+
+const addForm = document.querySelector('#add-task');
```

Comment:
Your index.js file is quite feature-rich. I think it would be better if you only have event listeners in this file and have a separate file for the functions instead.

Reply:
@tkz96 I tried not to overlide the functionality of `this` because i was using `class`. but I am going to work on it so that the index.js will be for event listeners if that wat you meant.